etc/nextcloudpi-config.d/nc-duckdns.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# DuckDNS installation on Raspbian for NextcloudPi
+#
+#
+# Copyleft 2017 by Courtney Hicks
+# GPL licensed (see end of file) * Use at your own risk!
+#
+
+ACTIVE_=no
+DOMAIN_=yourduckdnsdomain
+TOKEN_=your-duckdns-token
+INSTALLDIR=duckdns
+INSTALLPATH=/etc/$INSTALLDIR
+CRONFILE=/etc/cron.d/duckdns
+DESCRIPTION="Free Dynamic DNS provider (need account from https://duckdns.org)"
+
+install() { :; }
+
+
+configure() 
+{
+
+   if [[ $ACTIVE_ == "yes" ]]; then
+      mkdir $INSTALLPATH 2> /dev/null
+      # Creates duck.sh script that checks for updates to DNS records
+      touch $INSTALLPATH/duck.sh
+      touch $INSTALLPATH/duck.log
+      echo -e "echo url=\"https://www.duckdns.org/update?domains=$DOMAIN_&token=$TOKEN_&ip=\" | curl -k -o $INSTALLPATH/duck.log -K -" > $INSTALLPATH/duck.sh
```

Comment:
the field `ip=` is empty, is that correct?

Seems like this downloads a bash script from duckDNS? correct?

Reply:
There is no need to enter a value for "ip=" in the URL provided by DuckDNS, they handle all of that for you

The download I believe is to just grab OK or KO from them, which will tell you that DuckDNS is working as it should or not